kalite/django_cherrypy_wsgiserver/cherrypyserver.py

```diff
@@ -158,13 +158,13 @@ def port_is_available(host, port):
 
 
 def run_cherrypy_server(host="127.0.0.1", port=None, threads=None, daemonize=False, pidfile=None, autoreload=False, startuplock=None):
-    port = port or getattr(settings, "PRODUCTION_PORT", 8008)
+    port = port
```

Comment:
Actually it's inside a function that's not being called from anywhere. Will clean this up instead.